src/modules/mavlink/mavlink_receiver.h

```diff
@@ -246,32 +252,37 @@ class MavlinkReceiver
 	orb_advert_t _transponder_report_pub{nullptr};
 	orb_advert_t _visual_odometry_pub{nullptr};
 
-	static constexpr int _gps_inject_data_queue_size{6};
-
 	uORB::Subscription _actuator_armed_sub{ORB_ID(actuator_armed)};
 	uORB::Subscription _control_mode_sub{ORB_ID(vehicle_control_mode)};
 	uORB::Subscription _vehicle_attitude_sub{ORB_ID(vehicle_attitude)};
 
+	static constexpr int _gps_inject_data_queue_size{6};
+
+	static constexpr unsigned int MOM_SWITCH_COUNT{8};
+
 	int _orb_class_instance{-1};
+	int _param_sub{-1};
```

Comment:
This never gets initialized. You can use `uORB::Subscription _param_update_sub{ORB_ID(parameter_update)};` now.

Reply:
Thanks @bkueng !  Fixed up!